plugins/nf-validation/src/main/nextflow/validation/SamplesheetConverter.groovy

```diff
@@ -80,7 +80,7 @@ class SamplesheetConverter {
         }
         else {
             Path fileSamplesheet = Nextflow.file(samplesheetFile) as Path
-            samplesheetList = fileSamplesheet.splitCsv(header:containsHeader ?: ["empty"], strip:true, sep:delimiter, quote:'"')
+            samplesheetList = fileSamplesheet.splitCsv(header:containsHeader ?: ["empty"], strip:true, sep:delimiter, quote:'\"')
```

Comment:
oh so that wasn't working properly before?

Reply:
Yes that was working :), this part was for the samplesheet conversion but somehow we didn't add it for the validation :grin:

Reply:
Oh I see